sdk/test/trace/tracer_test.cc

```diff
@@ -113,7 +113,7 @@ TEST(Tracer, ToMockSpanExporter)
   ASSERT_EQ("span 2", spans_received->at(0)->GetName());
   EXPECT_TRUE(spans_received->at(0)->GetTraceId().IsValid());
   EXPECT_TRUE(spans_received->at(0)->GetSpanId().IsValid());
-  EXPECT_FALSE(spans_received->at(0)->GetParentSpanId().IsValid());
+  EXPECT_TRUE(spans_received->at(0)->GetParentSpanId().IsValid());
```

Comment:
Do we also check the values of trace_id and parent_span_id match?

Reply:
Added tests for this.